src/lib/internal/actions/interact-outside/types.ts

```diff
@@ -0,0 +1,7 @@
+export type InteractOutsideEvent = PointerEvent | MouseEvent | TouchEvent;
+
+export type InteractOutsideConfig = {
+	onInteractOutside?: (e: InteractOutsideEvent) => void;
```

Comment:
If the other one is start, what is this one? End? Maybe some JSDoc here would help clear things up